tests/lambda_endpoints/test_indexer_lambda.py

```diff
@@ -132,6 +132,13 @@ def populate_db(test_engine):
         yield session
 
 
+@pytest.fixture()
+def set_env():
+    # Set the environment variable
+    os.environ["S3_DATA_BUCKET"] = "data-bucket"
+    return ""
```

Comment:
We should set this with setenv:
https://docs.pytest.org/en/latest/reference/reference.html#pytest.MonkeyPatch.setenv
Do you want to put this in a conftest.py file for all tests?

```suggestion
@pytest.fixture(autouse=True)
def set_env(monkeypatch):
    # Set the environment variable
    monkeypatch.setenv("S3_DATA_BUCKET", "test-data-bucket")
```

Reply:
oh ok. Thank you!